### DISCUSSION TITLE
Any issues with ducted?

### DISCUSSION THREAD

**Discussion by bobbydavid** [gysmo38/mitsubishi2MQTT#247] [Q&A]
I have a ducted Mitsubishi air handler. Obviously there are no vanes to control... but otherwise, are there any other issues I should consider before diving into this? Thanks!

**Comment by floatplane** [comment]
I have a ducted air handler and I'm using this successfully. It is slightly more challenging, assuming your setup is like mine, with one of these remotes on the wall:

![image](https://github.com/gysmo38/mitsubishi2MQTT/assets/101196/bb41a336-bd37-46b0-a677-b1b6aa9db92b)

The main issues I ran into are:
- The ESP32 board needs to be connected into the same place as the remote receiver, so I don't think you can have both. I ended up taking the remote out of the system.
- If you lose the remote from your system, then you lose remote temperature sensing and the air handler will measure the temperature of the return air by default (which tends to be cold, which makes your heater run too much, so you don't want that).

I ended up using an Ecobee thermostat in my system which is not wired to any hardware, and connected that to a Home Assistant instance (which in turn publishes state to Node-RED for the logic - I find the scripting interface of HA obtuse at best). Every ~10 seconds I do the following:

- gather the target temperature and current temperature measurement from the Ecobee
- use MQTT to write those values to the ESP board

I have the same setup for the minisplits in my system, so that makes things easy for anyone to control temperature from the wall. From the Ecobee I don't have control of swing, fan speed, etc but I just leave those on "auto" anyway. I also get remote control, scheduling etc through the Ecobee app.

I ended up making myself a dashboard in HA so I could track the state of all these virtual thermostats. If something goes wrong (heater running non-stop for example) this gives me an idea of where to start looking.

<img width="1082" alt="image" src="https://github.com/gysmo38/mitsubishi2MQTT/assets/101196/7227d9a4-c720-46ea-947e-d96d93a3f711">

It took me a while to make this solid enough to be happy but now I don't have to really think about it, aside from rebooting the ESP boards once a month (I still haven't updated to a version that includes #226, so I lose the MQTT connection if I let the boards run long enough).

**Comment by bobbydavid** [comment]
Thanks for the response! This is really helpful. No, I don't have that
remote (I think it's the MHK2?). My installers gave me an ecobee instead,
it runs into an adapter, and that adapter plugs into CN105 in my air
handler.

According to the internet, the MHK2 receiver allows you to daisy chain the
CN105 connector, which is intended for someone to add the Kumo Cloud
controller (plug the Kumo Cloud controller into the MHK2 receiver, which in
turn plugs into the air handler board.

I'm currently considering the following:

 1. Get the MHK2 and plug it into CN105 instead of the Ecobee adapter.
 2. Daisy-chain an Arduino into the CN105 of the MHK2's receiver (using
this project).

I'd appreciate any feedback. I'll report back once I try it out.

**Reply by floatplane** [reply]
Yep, that's the MHK2 - it was what my installers gave me. I am curious to hear if you can daisy-chain into it! I can't actually remember what I did with the receiver once I removed it or I would take another look. On the other hand, I *hated* the UX of the MHK2 so I was strongly motivated to get it out of my system 😂 

I also just found a [CN105 splitter](https://gotductless.com/products/airzone-aidoo-cn105-splitter-for-mitsubishi) that I don't think existed a year ago...at least, I never saw it before. So maybe you could daisy-chain your ecobee setup and save the cost of the MHK2?

I am curious to hear how it goes, good luck!

**Comment by bobbydavid** [comment]
That's very interesting. I wonder how the CN105 splitter works. Does it
allow one device to change the temp that's set on the downstream device? I
would be fine with the UX of the MHK2 if I effectively never had to touch
it, but would still benefit from its improved ability to manage my variable
blower and inverter :)
